public/index.html

```diff
@@ -195,6 +195,9 @@
 
 .navbar-default .navbar-toggle {
   border: none; }
+
+#playground-v3-link {
```

Comment:
TBH not sure if this one should be inhere or moved to `style-3.css`.
Placed it here because this is only file that mentions this ID in script (and it was easier to do without local clone, but can move it there if needed)

Reply:
This is fine!